hack/release/process-changelog.go

```diff
@@ -1,108 +0,0 @@
-// Copyright 2021 Antrea Authors
```

Comment:
cc @antoninbas to check if it's ok to be removed.

Reply:
no issue on my side